lib/basedriver/commands/session.js

```diff
@@ -2,16 +2,26 @@ import _ from 'lodash';
 import log from '../logger';
 import { errors } from '../../mjsonwp';
 import UUID from 'uuid-js';
+import { processCapabilities } from '../capabilities';
 
 let commands = {};
 
-commands.createSession = async function (caps) {
+commands.createSession = async function (desiredCapabilities, requiredCaps, capabilities) {
```

Comment:
I think the changes are 'additive'. 
* `caps` is renamed to `desiredCapabilities` to distinguish it from `capabilities`
* `requiredCaps` is unused, it gets passed in by routes.js but doesn't do anything
* `capabilities` is a new parameter